backend/chainlit/oauth_providers.py

```diff
@@ -239,12 +239,31 @@ async def get_user_info(self, token: str):
                 # Ignore errors getting the photo
                 pass
 
```

Comment:
```suggestion
user_groups = []
```
If the call fails, user_groups will not be defined